behavioral-questions.md

```diff
@@ -4,16 +4,24 @@ Answer the following questions. Use the Nugget + STAR method for each answer.
 
 1. Why did you choose a career in software development?
 
-- Nugget:
-- Situation:
-- Task:
-- Action:
-- Result:
+- Nugget: I want to secure better career options for myself.
```

Comment:
When responding to this question, consider some of the attributes that software development has that interest you.  Interviewers like to hear deep answers that show that your interests align with theirs.  For instance, if you enjoy problem solving, or puzzles, or expanding on how you love learning could make this answer stand out.

Reply:
copy i will def add that